docs/source/plugin/time.rst

```diff
@@ -0,0 +1,211 @@
+==========
+About time
+==========
+
+Your plugin may want to display dates and times in messages. For that, you can
+always count on the :mod:`datetime` built-in module. However, what if you would
+like to respect the date-format for a given user or a given channel? Various
+functions of :mod:`Sopel<sopel.tools.time>` can help you with that:
```

Comment:
```suggestion
like to respect the date-format for a given user or a given channel? Functions
of :mod:`sopel.tools.time` can help you with that:
```

My goal here is to avoid referencing `Sopel` in monospace, because of how Sphinx deals with `:mod:` references:
![image](https://user-images.githubusercontent.com/164140/150650917-d2a2c016-1049-4dc9-9e68-eaf43301d6e0.png)